### PR TITLE
Adds login screen POC

### PR DESCRIPTION
**Pull request by gregnr**
We have no login screen and the user needs the ability to authenticate. This PR creates a very basic POC that will allow the user to login to the application.

This PR did not focus on styling - that will be addressed in another PR.

### CODE REVIEW

**Comment by Gabriel-Dalmoro** [comment]
Write better code pls

File: login-screen.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<html>
+
+<head>
+  <title>Login screen</title>
```
Comment (Gabriel-Dalmoro) [inline login-screen.html]: Change this to 'The Amazing Login Screen'

Comment (danibarker) [reply login-screen.html]: Looks cool

Comment (danibarker) [reply login-screen.html]: Can you also add a ! or 4?

**Comment by vinceiannelli** [comment]
Don't forget about it.

**Comment by yanliu1111** [comment]
wow here is not fun!

File: login-screen.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+</head>
+
+<body>
+  Login screen. With more functionality.
+</body>
```
Comment (vinceiannelli) [inline login-screen.html:9]: Shouldn't this be `</buddy>` ?

```js
const hello = 'hello'
```

File: login-screen.html (hunk 1 above)
Comment (gregnr) [reply login-screen.html]: @Gabriel-Dalmoro Really great point. Didn't think of that. That will make this much better.

Comment (yanliu1111) [reply login-screen.html]: Yes, never see this kind before!

File: login-screen.html (hunk 2 above)
Comment (Gabriel-Dalmoro) [reply login-screen.html:9]: I think you mean </booty>

Comment (gregnr) [reply login-screen.html:9]: Great point, I thought about that earlier, but I think `</body>` is actually what want here for reasons...

**Review by Gabriel-Dalmoro — APPROVED** [review]
# Amazing job. 

I've never seen such incredible code.
> "WOW"
image

image

File: login-screen.html (hunk 2 above)
Comment (danibarker) [reply login-screen.html:9]: I disagree.  ```
<buddy />``` is cooler

**Review by vinceiannelli — APPROVED** [review]

**Pull request merged**